src/View/Helper/DatatableHelper.php

```diff
@@ -669,7 +669,17 @@ protected function processActionLink(array $link): string
         case Datatables::LINK_TYPE_POST:
             $output = new \CakeDC\Datatables\View\Formatter\Link\PostLink($this, $link);
             break;
-
+			case Datatables::LINK_TYPE_CUSTOM:
+				if (!is_callable($link['formatter'] ?? null)) {
+					throw new \OutOfBoundsException("Please specify a custom formatter");
+				}
+				$output = new $link['formatter']($this,$link);
+
+				if (method_exists($output, 'link')){
```

Comment:
missing `!` as we need to throw exception if the link method does not exist

Reply:
Copied it wrong, my bad